packages/slate-react/src/components/content.js

```diff
@@ -144,6 +144,11 @@ class Content extends React.Component {
     const { isBackward } = selection
     const window = getWindow(this.element)
     const native = window.getSelection()
+
+    // .getSelection() can return null in some cases
+    // https://bugzilla.mozilla.org/show_bug.cgi?id=827585
+    if (!native) return
```

Comment:
Hmm, won't this prevent the DOM selection from updating to match the editor correctly? I wonder if maybe there should be some `null` checks further down to make sure the rest of the logic gets executed.

Reply:
@ericedem do you have any suggestion as to tests that I could perform to de-risk this PR?

Reply:
Nope, I think unfortunately `content.js` doesn't really have any tests yet. I'm not familiar enough with the codebase to give a recommendation.